src/pages/Explore_Regions.py

```diff
@@ -110,11 +127,35 @@ def plot_comp(m_route, cause, df):
         cause_of_death_input)]
 
 
-if len(route_input) > 0:
-    for i in route_input:
-        # st.map(migrantdf)
-        plot_deaths_season(i, df1)
-        plot_deaths_cause(i, cause_of_death_input, df2)
+if route_input:
+    route_s= route_input[0]
+    plot_deaths_month(route_s,cause_of_death_input,df_wout_cause,df_w_cause)
+    plot_deaths_and_survivors_month(route_s,cause_of_death_input,df_wout_cause,df_w_cause)
+    plot_deaths_season(route_s, df1)
+    plot_deaths_cause(route_s,cause_of_death_input, df2)
 
 if len(route_input) > 0 and len(cause_of_death_input) > 0:
     plot_comp(route_input, cause_of_death_input, df2)
+
+
+
+if route_input:
+
+    mdf = df[df['Migration route'] == route_s]
+    migrantdf = mdf.groupby(['Migration route'])[
+    'Cause of Death'].agg(pd.Series.mode).reset_index()
+    st.metric(f'Most common cause of death',migrantdf['Cause of Death'].iloc[0])
+    col1,col2= st.columns(2)
+    route_s= route_input[0]
+    worst_month = mdf.groupby(['Migration route','date'])['Total Number of Dead and Missing'].sum().reset_index().sort_values(by='Total Number of Dead and Missing',ascending=False)
+    df_d_s = mdf.groupby(['Migration route'])[['Total Number of Dead and Missing','Number of Survivors']].sum().reset_index()
+    
+    with col1:
+        st.metric(f'Total number of Dead and Missing',df_d_s['Total Number of Dead and Missing'].iloc[0])
+    with col2:
+        st.metric(f'Number of Survivors',df_d_s['Number of Survivors'].iloc[0])
+    col3,col4 = st.columns(2)
+    with col3:
+        st.metric(f'Deadliest event date',worst_month['date'].dt.strftime('%Y-%m').iloc[0])
+    with col4:
+        st.metric(f'Deadliest event number of deaths',worst_month['Total Number of Dead and Missing'].iloc[0])
```

Comment:
These metrics at the bottom are really great. 

Maybe let's add the second cause of death if the first is "mixed/unknown"?
<img width="638" alt="Screen Shot 2022-10-19 at 09 34 08" src="https://user-images.githubusercontent.com/53492091/196614534-75021d3b-33f3-4da7-bd0f-765a1166f972.png">